tests/test-smtlib-reader.cpp

```diff
@@ -112,63 +100,6 @@ TEST_P(IntReaderTests, QF_UFLIA_Smt2Files)
   }
 }
 
-TEST_P(BitVecReaderTests, QF_UFBV_Smt2Files)
```

Comment:
There are some removals in this file that seem unrelated to the PR. Are those on purpose?

Reply:
No those are not on purpose. I don't remember making any changes to that file, and tried at least to remove it from the pr.